scheduler/src/cook/mesos/util.clj

```diff
@@ -717,7 +717,7 @@
         schedule (tp/periodic-seq start-time frequency)
         chime-fn (fn [_]
                    (when @mesos-leadership-atom
-                     (let [age (-> -7 t/days t/from-now tc/to-date)]
+                     (let [age (-> -7 t/days t/from-now tc/to-date-time)]
```

Comment:
Since we've been burned once by this, can we have a test for it? One approach is to use `util/chime-at-ch` and have a channel passed in so you can manually trigger it in the test.

Reply:
I didn't see an example of that kind of use, but I tested it this way. Look ok?